keyboards/kuritakey/m0110/readme.md

```diff
@@ -0,0 +1,16 @@
+# Kuritakey M0110 PCB
+
+![M0110](https://i.imgur.com/e6VVWjQ.png)
+
+M0110 Rejuvenation PCB.
+
+* Keyboard Maintainer: [charlie-hotel](https://github.com/charlie-hotel/)
+* Hardware Supported: Kuritakey M0110 PCB
+* Hardware Availability: Group buy, now closed - was at kuritakey.store.
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```